.github/workflows/format.yml

```diff
@@ -0,0 +1,23 @@
+name: Scalafmt
+
+permissions: read-all
+
+on:
+  pull_request:
+    branches: ['**']
+
+jobs:
+  build:
+    name: Code is formatted
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout current branch (full)
+        uses: actions/checkout@v2
```

Comment:
```suggestion
        uses: actions/checkout@v3
```

Reply:
Ah, nice, thank you!
Will merge after this update.